rmgpy/data/kinetics/family.py

```diff
@@ -1181,6 +1181,49 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
                     atom.label = '*' + str(identicalCenterCounter)
             if identicalCenterCounter != 2:
                 raise KineticsError('Unable to change labels from "*" to "*1" and "*2" for reaction family {0}.'.format(label))
+        # Hardcoding of reaction family for peroxyl disproportionation
+        # *1 and *2 have to be changed to *3 and *4 for the second reactant
+        elif label == 'peroxyl_disproportionation' and forward:
+            identicalCenterCounter1 = identicalCenterCounter2 = 0
+            for atom in reactantStructure.atoms:
+                if atom.label == '*1':
+                    identicalCenterCounter1 += 1
+                    if identicalCenterCounter1 > 1:
+                        atom.label = '*3'
+                elif atom.label == '*2':
+                    identicalCenterCounter2 += 1
+                    if identicalCenterCounter2 > 1:
+                        atom.label = '*4'
+            if identicalCenterCounter1 != 2 or identicalCenterCounter2 != 2:
+                raise KineticsError('Unable to change labels from "*1" and "*2" to "*3" and "*4" for reaction family {0}.'.format(label))
+        # Hardcoding of reaction family for bimolecular hydroperoxide decomposition
+        # *2 has to be changed to *4 for the second reactant and '*1' has to be
+        # changed to '**' (only used to be able to relabel in the reverse direction)
+        # *3 has to be changed to '*' for the first reactant (also for reverse)
+        elif label == 'bimolec_hydroperoxide_decomposition' and forward:
+            identicalCenterCounter1 = identicalCenterCounter2 = identicalCenterCounter3 = 0
+            for atom in reactantStructure.atoms:
+                if atom.label == '*1':
+                    identicalCenterCounter1 += 1
+                    if identicalCenterCounter1 > 1:
+                        atom.label = '**'
```

Comment:
Using *# would imply that the atoms are undergoing bond changes in the reaction, but they're not. So I'm just calling them * and ** because I have to be able to distinguish them when applying the template in the reverse direction.